### ISSUE TITLE
Are pmatrix, bmatrix, vmatrix, etc. available in sagetex?

### ISSUE THREAD

**Issue by A1exN** [dandrake/sagetex#46]
When I type something like:
sage: m=matrix([[1,2],[3,4]])
and use \sage(m) in my latex document, it returns (not explicitly) the following text:

\left(\begin{array}{rr}
1 & 2 \\
3 & 4
\end{array}\right)

But there is a way it can return instead of that a text (depending of the type matrix I choose) like this:

\begin{pmatrix}
1 & 2 \\
3 & 4
\end{pmatrix}

or simply this:

\begin{array}{rr}
1 & 2 \\
3 & 4
\end{array}

In other words, are different types of matrices available in sagetex?

**Comment by dimpase** [comment]
I think sagetex takes the LaTeX representaton from Sage itself, namely, from [here](https://github.com/sagemath/sage/blob/10ed24e3ed0b5440624d5f1a9b18d2f8f89d8126/src/sage/matrix/matrix0.pyx#L2084)

E.g.
```
sage: m=matrix([[1,2,3],[4,5,6]])
sage: m._latex_??
```
will show docs and code - and `array` is hardcoded there.
One can certainly improve this, but this ought to be done in Sage, not here.
